common/components/UI/RealEstateComponents/RealEstateModal/RealEstateForm/index.tsx

```diff
@@ -9,19 +9,22 @@ import { IExtendedRealestate } from '@common/api/realestateApi/realestate.api.ty
 
 interface Props {
   form: FormInstance<any>
-  currentRealEstate: IExtendedRealestate
+  currentRealEstate?: IExtendedRealestate
+  editable?: boolean
+  disabled?: boolean
 }
 
-const RealEstateForm: FC<Props> = ({ form, currentRealEstate }) => {
-  const initialValues = useInitialValues(currentRealEstate)
+const RealEstateForm: FC<Props> = ({
+  form,
+  currentRealEstate,
+  editable = true,
+  disabled = false,
```

Comment:
Form `disabled` is not require in our cases. All situations now can be handled by `editable` only